tortoise/backends/base/client.py

```diff
@@ -7,6 +7,18 @@
 from tortoise.backends.base.schema_generator import BaseSchemaGenerator
 
 
+class Capabilities:
+    def __init__(self, dialect: str, *, connection: dict = {}, **kwargs) -> None:
+        self.dialect = dialect
+        self.connection = connection
+
```

Comment:
Thought experiment here: if we want to know whether client X has capability Y, we'd add a flag here for the base case, e.g. `self.supports_y = False`, and set it to `True` on clients that do support it?

Reply:
Yes, that's the idea :-)